assets/app/scripts/controllers/buildConfig.js

```diff
@@ -103,6 +102,12 @@ angular.module('openshiftConsole')
           updateFilterWarning();
           LabelFilter.addLabelSuggestionsFromResources($scope.unfilteredBuilds, $scope.labelSuggestions);
           LabelFilter.setLabelSuggestions($scope.labelSuggestions);
+
+          // Sort now to avoid sorting on every digest loop.
+          $scope.orderedBuilds = orderByDate($scope.builds, true);
+          if ($scope.orderedBuilds.length) {
+            $scope.latestBuild = $scope.orderedBuilds[0];
```

Comment:
else $scope.latestBuild = null? in theory there could have been one build, and then that build got deleted, in which case latestBuild shouldnt be set anymore